docs/INSTALL.K8S.md

```diff
@@ -26,12 +26,20 @@ nohup ./kube-controller-manager --master=127.0.0.1:8080 \
 nohup ./kube-scheduler --master=127.0.0.1:8080 --v=2 2>&1 > /dev/null &
 ```
 
-On the minions, start the kubelet specifying that the network plugin is of type
-CNI. e.g:
+On the minions, you need to download a few upstream CNI plugins
+
+```
+mkdir -p /opt/cni/bin && cd /opt/cni/bin
```

Comment:
I am supposed to be the pedant one here.
Please don't make me redundant.

The same applies to the whole document, so maybe a note about using a root shell might be added at the top